Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM nginx:1.21.0
+
+
+# Install necessary packages 
+RUN apt-get update -y && apt-get install -y apt-transport-https ca-certificates wget \
+    curl gnupg2 lsb-release sl mlocate dos2unix bash-completion openssl
```

Comment:
Please, remove the packages which are not necessary, to avoid increasing the docker size. For instance, bash-completion and dos2unix should not be needed, as well as either curl or wget could be dropped. Also the other ones should be checked if necessary.